agent-packaging/packaging-scripts/build_packages_deb.sh

```diff
@@ -15,22 +15,21 @@
 
 set -e
 
-function exit_error
-{
-  echo "build failed"
-  exit 1
-}
+BASE_DIR=${PWD}
 
-trap exit_error ERR
+echo "started build..."
+GCS_PATH="osconfig-agent-package"
```

Comment:
i think target env variables can probably be put in `prow->config.yaml`

Reply:
removed the variable, using it directly in the script as it is very specific to osconfig-agent. in case in future we feel like the script can be made generic, we will refactor the script